docs/source/conf.py

```diff
@@ -108,6 +108,15 @@
         '--project-url=../index.html',
         f'{_pydoctor_root}/pydoctor',
         ] + _common_args,
+    'rtd_theme_api': [
```

Comment:
very minor comment.

can we just use the epytext example to showcase the rtd theme?

so that we don't need yet another example.

The idea is that we have the real API docs for pydoctor itself using the standard theme and we have epydoc_demo with RTD theme.

Reply:
I've done what you suggested.